trade-finance-manager-api/src/v1/graphql-mappings/mappings/facilities/mapFacility.api-test.js

```diff
@@ -0,0 +1,148 @@
+const mapFacility = require('./mapFacility');
+const { formattedNumber } = require('../../../../utils/number');
+const mapFacilityProduct = require('./mapFacilityProduct');
+const mapFacilityType = require('./mapFacilityType');
+const mapFacilityStage = require('./mapFacilityStage');
+const mapFacilityValue = require('./mapFacilityValue');
+const mapBankFacilityReference = require('./mapBankFacilityReference');
+const mapGuaranteeFeePayableToUkef = require('./mapGuaranteeFeePayableToUkef');
+const mapBanksInterestMargin = require('./mapBanksInterestMargin');
+const mapFirstDrawdownAmountInExportCurrency = require('./mapFirstDrawdownAmountInExportCurrency');
+const mapFeeType = require('./mapFeeType');
+const mapFeeFrequency = require('./mapFeeFrequency');
+const mapDates = require('./mapDates');
+
+const MOCK_DEAL = require('../../../__mocks__/mock-deal');
+
+describe('mapFacility', () => {
+  const mockTfmFacility = {
+    ukefExposure: '1,234.00',
+    ukefExposureCalculationTimestamp: '1606900616651',
+    facilityValueInGBP: '12345',
+  };
+
+  const mockDealDetails = MOCK_DEAL.details;
+
+  const mockCoverEndDate = {
+    'coverEndDate-day': '01',
+    'coverEndDate-month': '02',
+    'coverEndDate-year': '2021',
+  };
+
+  const mockCoveredPercentage = '10';
+
+  const mockCurrency = {
+    text: 'GBP - UK Sterling',
+    id: 'GBP',
+  };
+
+  const mockFacilityValue = '12345.00';
+  const mockType = 'Bond';
+  const mockFacilityStage = 'Unissued';
+
+  const mockFacility = {
+    _id: '12345678',
+    dealId: '100200300',
+    ukefFacilityId: '0040004833',
+    type: mockType,
+    ukefFacilityType: mockType,
```

Comment:
how sure are we that these mock types are the same (type/ukefFacilityType)?

Reply:
As these are existing and due to the number of changes in this PR, we shall leave this as is